server/src/main/java/org/apache/iotdb/db/conf/IoTDBConfigCheck.java

```diff
@@ -203,6 +203,13 @@ public void checkConfig() throws IOException {
       }
       // rename tmpLogFile to mlog
       FileUtils.moveFile(tmpMLogFile, mlogFile);
+
+      File oldMLogFile = SystemFileFactory.INSTANCE.getFile(SCHEMA_DIR + File.separator
+          + MetadataConstant.METADATA_OLD_LOG);
```

Comment:
Line 190 to line 205 is the part of upgrading v0.9 mlog to v0.10. Since the `MetadataConstant.METADATA_LOG` has changed, I don't think that upgrading will work correctly. Should we remove that?

Reply:
yes, it's no meaning.